### DISCUSSION TITLE
Emgu.cv.runtime.uwp

### DISCUSSION THREAD

**Discussion by VietHung-CHV** [emgucv/emgucv#913] [Q&A]
I don't find Emgu.cv.runtime.uwp on nuget. I want to build app run on Hololens 2, so which is the namespace I need to build app? Emgu.cv.runtime.uwp or Emgu.cv.runtime.windows?

**Comment by canmingh** [comment]
Emgu.cv.runtime.uwp is only available under the commercial license. Once license is purchased, you can find it in our commercial nuget repository.

Hololens 2 use the UWP SDK and will need to use the Emgu.CV.runtime.uwp nuget package instead of the Emgu.CV.runtime.windows nuget package.
